test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -424,6 +437,16 @@ private string GetProfilerProjectBin()
                 EnvironmentTools.GetPlatform().ToLower());
         }
 
+        private string GetProfilerProjectNonWindowsBin()
+        {
+            return Path.Combine(
+                GetSolutionDirectory(),
+                "src",
+                "Datadog.Trace.ClrProfiler.Native",
+                "build",
+                "bin");
+        }
+
```

Comment:
Is it possible to make the profiler build to the same dir in both Windows and Non-Windows? I feel like this discrepancy is going to cause other issues when we try and improve the build pipeline in general 🙁

Reply:
Ok, no problem, right now we simulate the windows folder structure: https://github.com/DataDog/dd-trace-dotnet/blob/tony/macos-integration-tests-fix/build/docker/Datadog.Trace.ClrProfiler.Native.sh#L15-L16

Reply:
done in dd94206